src/util/hass-util.html

```diff
@@ -63,12 +65,23 @@
   Object.keys(attributes).forEach(function (key) {
     customEl[key] = attributes[key];
   });
+  customEl.isVisible = true;
```

Comment:
Yes, adding it to more-info-content.html sounds like the way to go. We wouldn't need `resetDynamicContent` - `dynamicContentUpdater` could just be used?